north/src/api.rs

```diff
@@ -19,7 +19,7 @@ use std::{collections::HashMap, path::PathBuf};
 
 pub type Name = String;
 pub type MessageId = String; // UUID
-pub use npk::archive::RepositoryId;
+use npk::archive::RepositoryId;
```

Comment:
Rustfmt should move this to the above use block.

Reply:
it does not, but i can move it :)